cache/compressed_secondary_cache.h

```diff
@@ -44,6 +46,16 @@ class CompressedSecondaryCacheResultHandle : public SecondaryCacheResultHandle {
 
 // The CompressedSecondaryCache is a concrete implementation of
 // rocksdb::SecondaryCache.
+// When a block is firstly Lookup from CompressedSecondaryCache, we just
+// insert a dummy block into the primary cache (charging the actual size of the
+// block) and don’t erase the block from CompressedSecondaryCache. A
+// standalone handle is returned to the caller. Only if the block is hit again,
+// we erase it from CompressedSecondaryCache and add it into the primary cache.
+//
+// When a block is firstly evicted from the primary cache to
+// CompressedSecondaryCache, we just insert a dummy block (size 0) in
+// CompressedSecondaryCache. When the block is evicted again, it is treated as
```

Comment:
Probably should clarify "is evicted again". Only if it is evicted again before the dummy block is evicted from the cache, we insert it.